content/blog/2022-01-16-open-scientific-computing-ross/index.md

```diff
@@ -27,8 +34,10 @@ A common problem that prevents many researchers from routinely sharing their wor
+Providing scientists with easy access to opportunities to practice open science, so it becomes more and more like any other routine scientific commodity, is an important open science challenge that Galaxy tries to address. With just a web browser, and relatively little training, scientists quickly become productive. Training is built in, as self directed learning material in the [GTN](https://training.galaxyproject.org/), as part of the infrastructure for making everything as easy as possible for researchers. The actual open science experiments and analyses still depend on the skills of the researcher, but by using Galaxy, all the associated benefits of open scientific computing practice make the results readily available to the scientific community, for replication and repurposing.
+
+Exactly how that will change science is hard to predict. Negative impacts seem unlikely, other than on   commercial competitor profitability. The peer review process would probably be far more useful, if all the reviewers could readily share, replicate and scrutinise a transparent experimental analysis. It seems likely to speed up the validation of more reliable, replicable new methods and findings. That may improve scientific progress by more rapid accumulation of scientific trust in the reproducible, and quicker abandonment of findings that cannot be replicated. It will likely accelerate the overall rate of discovery, by making it much easier to repurpose the analysis computing already developed for related work. It will certainly make research more efficient in terms of adding additional value to previous analyses, particulary if previous work is easily discoverable and repurposable.
 
-One reason to [become involved](../2022-01-14-users-guide-to-contribution-ross) in the Galaxy project is to contribute to normalising open science computing. Exactly how that will change science is hard to predict but we can make some informed guesses. Negative outcomes seem unlikely, other than perhaps for  commercial competitor profits. It seems likely to speed up the validation of more reliable, replicable results. That may improve scientific progress by more rapid accumulation of scientific trust in the reproducible results, and quicker abandonment of findings that cannot be replicated. It will likely accelerate the overall rate of discovery, by making it much easier to repurpose the analysis computing already developed for related work. It will certainly make research more efficient in terms of adding additional value to previous analyses, particulary if previous work is easily discoverable and repurposable.
+If you've managed to read this far, you may want to learn more about opportunities to help make open science computing more accessible. If so, you are cordially invited to [become involved in the Galaxy project](../2022-01-14-users-guide-to-contribution-ross) in any way you can.
```

Comment:
Could this be a good place to link the contributing page too?

Reply:
Maybe...it's a little awkward having two parallel resources :) 
Let me ponder it.

Reply:
@beatrizserrano: ok, reverted. I think we are done for now. The ministry for truth is established.

Reply:
do we merge then?

Reply:
Both linked now - it's not pretty in some ways but they are different enough to be useful. The blog's not as exhaustive - maybe it needs merging but I'm over it for now.

Reply:
I think that they are addressing different audiences, depending on the reading preferences.

Reply:
@beatrizserrano: merge please. I'm done for now :)
Thanks!